pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -446,7 +445,7 @@ private BrokerResponseNative handleRequest(long requestId, String query, JsonNod
     }
 
     if (offlineBrokerRequest == null && realtimeBrokerRequest == null) {
-      if (pinotQuery.isExplain()) {
+      if (serverPinotQuery.isExplain()) {
```

Comment:
We should check the broker query instead of the server query here. Server query might be the subquery of the broker query, and it might not make sense to have EXPLAIN in the subquery